src/services/member/controller.ts

```diff
@@ -101,6 +113,48 @@ const controller: FastifyPluginAsync = async (fastify) => {
       });
     },
   );
+
+  fastify.post<{ Body: { email: string } }>(
+    '/current/email/change',
+    { schema: postChangeEmail, preHandler: isAuthenticated },
+    async ({ user, body: { email } }, reply) => {
+      const member = notUndefined(user?.member);
+      reply.status(StatusCodes.NO_CONTENT);
+      if (await memberService.getByEmail(buildRepositories(), email)) {
+        // Email adress is already taken, ignore the request
+        return;
```

Comment:
User does not get feedback if they try to use an email that is already in use ? I can understand that this would leak information of emails of other users. But how should the interface let the user know that it did not work ? They would wait on the email in the new mailbox for eternity.